### DISCUSSION TITLE
EP not Removing From Laptop

### DISCUSSION THREAD

**Discussion by Aadii8430** [valinet/ExplorerPatcher#1570] [General]
Hello, I uninstall explorer patcher and also delete its registry folder and also delete its folder from C , but still after restart my laptop, i see my taskbar showing like explorer patcher and the registry folder also back but suspecious thing when i go to control pannel to again uninstall it, I see there is no explorer patcher to uninstall 🥲🥲 , I don't know what to do now to remove it as its also not showing in setting installed apps 😐

Its just showing in window search and from their when i click uninstall, it redirect me to control panel and in control panel their is no explorer patcher, i tried to view hidden files and folders but still can't see in control panel. 

![IMG20230620124113](https://github.com/valinet/ExplorerPatcher/assets/63859896/3da9844e-0ffc-401f-baa0-ed52953ee64a)

**Comment by pyrates999** [comment]
Here's how to uninstall it: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

**Reply by Aadii8430** [reply]
Bro please chek post completely, i told that i already uninstalled like that , and its not showing their again now to be uninstalled again but its still exist in my system

**Reply by pyrates999** [reply]
How did you uninstall it?

**Comment by Aadii8430** [comment]
I uninstall it from control panel and from setting> apps and also delete
its registry and delete his all folders from C and other locations
completely

**Comment by pyrates999** [comment]
You may have to remove the short cut of it from the start menu then.  It's in ExplorerPatcher -> Properties (ExplorerPatcher).

**Comment by Aadii8430** [comment]
That also aready deleted

**Reply by pyrates999** [reply]
Might be cached by the start menu then.

**Comment by Aadii8430** [comment]
May be, if that the case then how to solve it?

**Reply by pyrates999** [reply]
Not sure as I don't run windows 11.  Google is your friend.

You can also just try opening it and see if it actually opens.  If it does, it means you didn't fully remove it.

**Comment by creeper-0910** [comment]
After installing ExplorerPatcher again, try running `ep_setup.exe /uninstall`.
